### PR TITLE
Update FAQs with mitid_core_client_error

### PR DESCRIPTION
**Pull request by nmoskaleva**

### CODE REVIEW

File: src/pages/verify/guides/faq.mdx [hunk 1]
```diff
@@ -19,4 +19,10 @@ The `Unknown Access Code` error can happen for one of two reasons when trying to
...
 
-You can use the copy application feature from test to production to ensure the settings are the same.
\ No newline at end of file
+You can use the copy application feature from test to production to ensure the settings are the same.
+
+## MitID Core Client error
+
+If you encounter the `mitid_core_client_error` while running test logins, it indicates issues with the MitID PP environment. 
+Please note that the MitID PP environment can occasionally experience instability, and MitID does not provide SLAs or status updates for this environment.
+Criipto does not have control over the MitID Test Tools.
```
Comment (mickhansen) [inline src/pages/verify/guides/faq.mdx:28]: Mention that if they are seeing issues with an app simulator user, they can try registering a TOTP authenticator instead.

Comment (nmoskaleva) [reply src/pages/verify/guides/faq.mdx:28]: Ok, good to know, thanks.

Comment (mickhansen) [reply src/pages/verify/guides/faq.mdx:28]: image
May want individual paragraphs.

Comment (nmoskaleva) [reply src/pages/verify/guides/faq.mdx]: Updated

**Pull request merged**